core/src/main/java/org/kohsuke/stapler/export/ExportConfig.java

```diff
@@ -14,6 +14,13 @@
 
     private ClassAttributeBehaviour classAttribute = ClassAttributeBehaviour.IF_NEEDED;
 
+    /**
+     * Interceptor used to check if given model object or it's property should be included in serialization
+     */
+    private ExportInterceptor exportInterceptor = ExportInterceptor.DEFAULT;
+
+    private boolean skipIfFail = false;
```

Comment:
Seems like this should be another method of `ExportInterceptor`, called I suppose when supplied with something which is not `@ExportedBean` or `CustomExportedBean`. Or at a higher level I would expect the whole handling of a bean to go through `ExportInterceptor`. IOW this looks like the wrong level of abstraction.